index.html

```diff
@@ -65,90 +65,98 @@ <h1 class="margin-bottom-0">Hey there. I’m Mihreteab.</h1>
+          <div class="desctiption">
+            <h4>Multi-Post Stories Gain+Glory</h4>
+            <ul>
+              <li><button type="button" class="btn-secondary" disabled>Ruby</button></li>
+              <li><button type="button" class="btn-secondary" disabled>css</button></li>
+              <li><button type="button" class="btn-secondary" disabled>JavaScript</button></li>
+              <li><button type="button" class="btn-secondary" disabled>html</button></li>
+            </ul>
+            <button type="button" class="btn btn-primary">See Project</button>
+          </div>
+        </article>
+        <article class="project">
+          <img src="./images/project4.jpg" alt="project" />
+          <div class="desctiption">
+            <h4>Multi-Post Stories Gain+Glory</h4>
+            <ul>
+              <li><button type="button" class="btn-secondary" disabled>Ruby</button></li>
+              <li><button type="button" class="btn-secondary" disabled>css</button></li>
+              <li><button type="button" class="btn-secondary" disabled>JavaScript</button></li>
+              <li><button type="button" class="btn-secondary" disabled>html</button></li>
+            </ul>
+            <button type="button" class="btn btn-primary">See Project</button>
+          </div>
+        </article>
+        <article class="project">
+          <img src="./images/project3.jpg" alt="project" />
+          <div class="desctiption">
+            <h4>Multi-Post Stories Gain+Glory</h4>
+            <ul>
+              <li><button type="button" class="btn-secondary" disabled>Ruby</button></li>
+              <li><button type="button" class="btn-secondary" disabled>css</button></li>
+              <li><button type="button" class="btn-secondary" disabled>JavaScript</button></li>
+              <li><button type="button" class="btn-secondary" disabled>html</button></li>
+            </ul>
+            <button type="button" class="btn btn-primary">See Project</button>
+          </div>
+        </article>
+        <article class="project">
+          <img src="./images/project1.jpg" alt="project" />
+          <div class="desctiption">
+            <h4>Multi-Post Stories Gain+Glory</h4>
+            <ul>
+              <li><button type="button" class="btn-secondary" disabled>Ruby</button></li>
+              <li><button type="button" class="btn-secondary" disabled>css</button></li>
+              <li><button type="button" class="btn-secondary" disabled>JavaScript</button></li>
+              <li><button type="button" class="btn-secondary" disabled>html</button></li>
+            </ul>
+            <button type="button" class="btn btn-primary">See Project</button>
+          </div>
+        </article>
+      </div>
     </section>
     <section class="about">
       <article class="about-me-description">
-        <h1>About me</h1>
+        <div class="top-right-background only-desktop">
+        </div>
+        <div class="top-left-background only-desktop">
```

Comment:
- [ ] _[OPTIONAL]_ It is advisable to attach the background images to the `section` or `article` tag, adding them as `div` tags is similar to an `img` tag.